charts/cortx/templates/_config.yaml.tpl

```diff
@@ -58,7 +58,7 @@ cortx:
       local: /etc/cortx
       config: /etc/cortx
     security:
-      ssl_certificate: /etc/cortx/solution/ssl/s3.seagate.com.pem
```

Comment:
This name could be anything.  "s3.seagate.com.pem" didn't seem right, so I changed it to a simple "cortx.pem".